pkg/backend/store.go

```diff
@@ -54,18 +54,26 @@ type store struct {
 	rev    revision.Revisioner
 	config *config.Config
 	t      *storage.Table
-	client storage.TableServiceClient
 }
 
 func NewBackend(c *config.Config) (Backend, error) {
-	revisioner := revision.NewRevisioner(c.Runtime.StorageTable)
+	storageTable := c.Runtime.StorageTable
```

Comment:
NewRevisioner(..) should operate on a configuration instance and make the decision internally. The details of using/not using separate table is all made inside revisioner. Avoid leaking the details of the revisioner outside of it.

Or alternatively (I vote that this is a better approach) the entire revisioner code use `revisionerTable` and the init runtime either points both TableClients to one instance (in case of same table) or two different instances (incase of separate tables)

Reply:
Done!